src/settings-ui/Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -1256,7 +1255,7 @@ Made with 💗 by Microsoft and the PowerToys community.</value>
     <value>Enable at least one plugin to get started</value>
   </data>
   <data name="Run_NotAllowedActionKeyword.Title" xml:space="preserve">
-    <value>This activation command is already in use by another plugin.</value>
+    <value>This activation command is forbidden or reserved for the use in other plugins.</value>
```

Comment:
This is more verbose.. and not not super clear to me, IMO.

Reply:
Was suggested a few commenys up here. Must have had a reason. After what condition is this text actually visible?

Reply:
This change can be reverted as this text will be removed/replaced in my PR. See https://github.com/microsoft/PowerToys/pull/19593#issuecomment-1209180786.

Reply:
Is there an _e.t.a._ for that pr? I would be unhappy if I revert this now, and my pr would make it in the next version and yours would (for whatever reason) not... Seems safer to me to let you just overwrite it.